src/VirtualAlexa.ts

```diff
@@ -224,11 +226,13 @@ export class VirtualAlexaBuilder {
             const utterances = SampleUtterances.fromFile(this._sampleUtterancesFile);
             model = new InteractionModel(schema, utterances);
         } else {
-            throw new Error("Either an interaction model or intent schema and sample utterances must be provided.");
+            model = InteractionModel.fromLocale(locale);
+            if (!model) {
```

Comment:
Add an additional sentence. "Alternatively, if you specify a locale, Virtual Alexa will look automatically check for the interaction model under the directory "./models" - e.g., "./models/en-US.json""